README.md

```diff
@@ -39,6 +39,8 @@ Once installed, you'll be able to launch OpenSpades from inside the desktop menu
 
 #### Building and installing from source
 GCC 4.9 / Clang 3.2 or later is recommended because OpenSpades relies on C++11 features heavily.
+   *On Debian-derived distributions, you can install GCC*:
+   ```sudo apt-get install build-essential```
```

Comment:
Good catch.

To clean up the instructions, please consider consolidating this with the package lists in the `Install dependencies` section below. If you are unsure about other distributions, just add a TODO comment wrapped in `<!-- ... -->`.

The sentence `GCC 4.9 / Clang 3.2 or later is recommended because OpenSpades relies on C++11 features heavily.` can go now that it's been a decade since the release of C++11. Instead, add a sentence below `Install dependencies` stating that a C++ compiler is required. This way, the readers can tell that a C++ compiler is already included in the shown package list.

<!-- "GCC 4.9 / Clang 3.2 or later is recommended because OpenSpades relies on C++11 features heavily." should be moved as well to the same section and maybe rephrased so as not to prompt the readers to install GCC or Clang separately. -->